pkgs/top-level/all-packages.nix

```diff
@@ -496,6 +496,8 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) AppKit;
   };
 
+  cmakerc = callPackage ../development/libraries/cmakerc { };
```

Comment:
Since we are now in the _by-name era_, can you move this file `development/libraries/cmakerc/default.nix` to `by-name/cm/cmakerc/package.nix`?

```suggestion
```